File: src/frontend/my-RacksStand/src/app/rack-boxes/shared/rack-box-model.ts

export class RackBoxModel {
  public Id: string;
  public Name: string;
  public Description: string;
  public RackId: string;
  public Row: number;
  public Column: number;
  public SecondaryStatus: number;
  public CompanyId: string;
  public Index: number;
  public CurrentSize: number;
}
export class InventoryModel {
  public Id: string;
  public Name: string;
  public Description: string;
  public AccessCode: string;
  public Status: number;
  public CompanyId: string;
  public CustomerId: string;
  public RackBoxId: string;
  public Index: number;
  public CreatedOn: Date;
  public CreatedBy: string;
  public UpdatedOn: Date;
  public UpdatedBy: string;
}

File: src/backend/BusinessLogic/Models/Core/RackBoxModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models.Core
{
    [Serializable]
    public class RackBoxModel
    {
        [Required]
        [MaxLength(32)]
        public string Id { get; set; }
        [Required]
        [MaxLength(100)]
        public string Name { get; set; }
        [MaxLength(250)]
        public string Description { get; set; }
        [Required]
        [MaxLength(32)]
        public string RackId { get; set; }
        public int? Index { get; set; }
        public byte SecondaryStatus { get; set; }
        public string CompanyId { get; set; }
        public int Row { get; set; }
        public int Column { get; set; }
        public int? CurrentSize { get; set; }
    }
}
